User.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace common;
+
+use yii\db\ActiveRecord;
+
+class User extends ActiveRecord
+{
+    public static function getCommentsForPosts()
```

Comment:
Set return type: array
```suggestion
    public static function getCommentsForPosts(): array
```